src/noise.jl

```diff
@@ -74,6 +99,14 @@ function PauliError(qubits,p)
     NoiseOp(PauliNoise(p), qubits)
 end
 
+function PauliError(q::Int, px, py, pz)
+    NoiseOp(PauliNoise(px,py,pz), [q])
```

Comment:
```suggestion
    NoiseOp(PauliNoise(px,py,pz), (q,))
```

no need to allocate a vector